resources/sdk/clisdkclient/templates/restclient.mustache

```diff
@@ -243,6 +282,93 @@ func authorize(c config.Configuration) (models.OAuthTokenData, error) {
 	return *oAuthTokenResponse, nil
 }
 
+// Starts the local server at the redirect URI
+func startLocalServerFunc(c config.Configuration, authChannel chan models.OAuthToken) {
+	var host string
+	if c.SecureLoginEnabled() {
+		host = "https://localhost"
+	} else {
+		host = "http://localhost"
+	}
+	address := strings.Replace(c.RedirectURI(), host, "", -1)
+	port := address[0:5]
```

Comment:
If you store RedirectURI as a URI (or at least parse it to one here), you won't have to come up with your own parsing logic to extract the different parts.